hyrisecockpit/api/app/workload/service.py

```diff
@@ -42,20 +42,14 @@ def create(cls, interface: WorkloadInterface) -> Optional[Workload]:
         Returns the created Workload.
         Returns None if a Workload with the given ID already exists.
         """
-        response = cls._send_message_to_dbm(
-            Request(header=Header(message="start worker"), body={})
+        response = cls._send_message_to_gen(
+            Request(header=Header(message="start workload"), body=dict(interface)),
+        )
+        return (
+            None
+            if response["header"]["status"] == 409
```

Comment:
The `WorkloadGenerator` can also return a 404 as seen in:
https://github.com/hyrise/Cockpit/blob/0adb411f8a75ab729cb9502f9ed96fc50d51a5e4/hyrisecockpit/workload_generator/generator.py#L97

We should probably handle this as well (separate PR).

We could fix this temporarily with something like
```suggestion
            if response["header"]["status"] != 200
```